modules/apps/27-interchain-accounts/host/keeper/keeper_test.go

```diff
@@ -217,8 +217,6 @@ func (suite *KeeperTestSuite) TestNewModuleQuerySafeAllowList() {
 	suite.Require().Contains(allowList, "/cosmos.bank.v1beta1.Query/AllBalances")
 	suite.Require().Contains(allowList, "/cosmos.staking.v1beta1.Query/Validator")
 	suite.Require().Contains(allowList, "/cosmos.staking.v1beta1.Query/Validators")
-	suite.Require().Contains(allowList, "/cosmos.circuit.v1.Query/Account")
-	suite.Require().Contains(allowList, "/cosmos.circuit.v1.Query/DisabledList")
```

Comment:
Why are these 2 removed?

Reply:
Circuit was removed from the app maintained in `ibctesting`

Reply:
Because the `testing` simapp (not the separate go.mod one we use for e2e tests) has been stripped down to reduce dependencies and as part of that the circuit module was removed. So these won't be in the allowList for unit tests anymore.

Reply:
Got it, thank you!